test/e2e/Command/AssertBackwardsCompatibleTest.php

```diff
@@ -184,7 +184,7 @@ public function testWillNotRunWithoutTagsNorSpecifiedVersions() : void
             $this->sourcesRepository
         );
 
-        self::assertSame(212, $check->run());
```

Comment:
This is a transient BC break coming from `symfony/console` - nothing to do about it though, as the code `212` was not supposed to be relied upon.